src/actions/smartWalletActions.js

```diff
@@ -1757,29 +1756,38 @@ export const estimateEnsMigrationFromArchanovaToEtherspotAction = () => {
       await dispatch(switchAccountAction(getAccountId(archanovaAccount)));
     }
 
-    const transactionsToEstimate = migratorTransactions.map(({
-      data,
-      to,
-    }) => ({ to, data, value: EthersBigNumber.from(0) }));
+    dispatch(setEstimatingTransactionAction(true));
+
+    let errorMessage;
+    let estimated;
+    try {
+      estimated = await archanovaService.estimateAccountRawTransactions(rawTransactions);
```

Comment:
Are we sure we need try catch here?

Reply:
Yes, in case estimation fails. This is isolated from regular estimation flow.